probe-rs/src/architecture/arm/core/armv7a.rs

```diff
@@ -616,6 +616,12 @@ impl<'probe> CoreInterface for Armv7a<'probe> {
         }
         Ok(breakpoints)
     }
+
+    fn fpu_support(&mut self) -> Result<bool, crate::error::Error> {
+        Err(crate::error::Error::Other(anyhow::anyhow!(
```

Comment:
Please file an issue after this goes in and tag me - I'll go clean up the implementations for v7a / v8a.